widgetssdk/src/main/java/com/glia/widgets/view/configuration/ChatHeadConfiguration.java

```diff
@@ -11,6 +11,9 @@
     private final Integer badgeTextColor;
     private final Integer backgroundColorRes;
 
+    private final Integer onHoldIcon;
```

Comment:
in the theme it's called `iconOnHold`, here it's a bit different. would be nice to keep it consistent throughout the codebase.